.github/workflows/pr-bundle.yml

```
@@ -0,0 +1,65 @@
+
+jobs:
+  build:
+    name: Build and Deploy Bundle
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
```

actionlint (1.7.9)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)